api/envoy/extensions/transport_sockets/http_11_proxy/v3/upstream_http_11_connect.proto

```diff
@@ -35,5 +35,5 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 //
 message Http11ProxyUpstreamTransport {
   // The underlying transport socket being wrapped.
-  config.core.v3.TransportSocket transport_socket = 1 [(validate.rules).message = {required: true}];
+  config.core.v3.TransportSocket transport_socket = 1;
```

Comment:
can you say more about why the underlying socket is no longer required?

Reply:
I thought the PR description covered the reasoning. If it's unclear, can you give me a pointer on what details you're looking for?

Reply:
So sorry, I missed the PR description.  That explains it, thanks!